api/dbConfig/seed.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO entries (title, name, story)
+VALUES
+    ('My Story', 'Deb'),
+    ('Story 2', 'Ajay');
```

Comment:
So over here you have mentioned that you have title name and story but when inserting values you have only given a title and name?